site/content/entry/yuzu-boxcat/index.md

```diff
@@ -35,37 +35,37 @@ Thanks to the efforts of our developer [DarkLordZach](https://github.com/DarkLor
 By emulating the `BCAT` service at a high-level, yuzu is able to intercept the game's calls to Nintendo's servers and reroute them to yuzu's servers.
 This means that games running on yuzu, will now check for new content on yuzu's servers instead of Nintendo's.
 
-And this allows us to add new in-game content, for games that use this service. 
+This allows us to add new in-game content for games that use this service. 
 For the inaugural run, our team members have added some cool content across different games which you can check out [here](https://yuzu-emu.org/help/feature/boxcat/).
-We will have new events frequently, so all users will have the chance have fun playing games on yuzu.
+We will have new events occasionally, so users will have the even more fun while playing games on yuzu.
 
 {{< imgs
     "./boxcat.png|Boxcat Settings Configuration"
 >}}
 
 ### When?
 
-We will be testing this feature in our Patreon preview builds first and it will be available to Canary users shortly.
+We will be testing this feature in our Patreon preview builds first and it will be available to Canary users shortly after.
```

Comment:
`shortly after` feels like _that_ day, or a day or two after

Maybe substitute `in the near future.`